src/all/mangaplus/src/eu/kanade/tachiyomi/extension/all/mangaplus/MangaPlusFactory.kt

```diff
@@ -20,3 +21,4 @@ class MangaPlusPortuguese : MangaPlus("pt-BR", "ptb", Language.PORTUGUESE_BR)
 class MangaPlusRussian : MangaPlus("ru", "rus", Language.RUSSIAN)
 class MangaPlusSpanish : MangaPlus("es", "esp", Language.SPANISH)
 class MangaPlusThai : MangaPlus("th", "tha", Language.THAI)
+class MangaPlusFrench : MangaPlus("fr", "fre", Language.FRENCH)
```

Comment:
```suggestion
class MangaPlusFrench : MangaPlus("fr", "fra", Language.FRENCH)
```

Should actually be `fra`, because it's the code the site is using.